packages/api/src/beacon/routes/validator.ts

```diff
@@ -1065,6 +1069,9 @@ export function getDefinitions(config: ChainForkConfig): RouteDefinitions<Endpoi
         },
       },
       resp: EmptyResponseCodec,
+      init: {
+        requestWireFormat: WireFormat.ssz,
```

Comment:
part of spec now https://github.com/ethereum/beacon-APIs/pull/508, we can enable ssz by default, worst case we get a 415 response from beacon node and fall back to JSON